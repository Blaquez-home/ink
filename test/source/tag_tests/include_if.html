---
---
{% include foo.html some_var="var_test" if site.name == 'Your New Jekyll Site' %}
{% include site.include_test some_var="var_test" if site.name != 'Your New Jekyll Site' %}
{% include site.include_test some_var="var_test" unless site.name == 'Your New Jekyll Site' %}
{% include site.include_test some_var="var_test" unless site.name == 'Your Stupid Website' %}
