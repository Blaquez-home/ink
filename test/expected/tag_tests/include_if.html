Testing Include var_test



Testing Include var_test

